gleam_webapp/candidate_app/views.py

```diff
@@ -365,6 +365,7 @@ def candidate_table(request):
         chgcentre_count=Count('rating', filter=Q(rating__cand_type='CC')),
         scintillation_count=Count('rating', filter=Q(rating__cand_type='S')),
```

Comment:
You define CC,S,P,AGN as variables in `models.py`, which gives you the ability to rename/describe them without making much changes to the code.

Would it not be better to use those variables here instead of their string representations?

Reply:
I had the same thought, but I don't think you can because they are strings and this needs function arguments. Unless there is some python magic I am unaware of